execution-engine/src/fs.rs

```diff
@@ -647,13 +775,23 @@ impl FileSystem {
         all_rights.insert(PathBuf::from("stderr"), Rights::all());
 
         rst.install_prestat::<PathBuf>(&all_rights)?;
+        //TODO include the correct parameter
+        let service: Box<dyn Service> = Box::new(PostcardService::new());
+        let mut services = Vec::new();
+        services.push((
+            "/services/postcard_string.dat",
+            rst.clone(),
+            Arc::new(Mutex::new(service)),
+        ));
+        rst.install_services(services)?;
         Ok(rst)
     }
 
     /// This is the ONLY public API to create a new FileSystem (handler).
     /// It return a FileSystem where directories are pre-opened with appropriate
     /// capabilities in related to `principal`.
     pub fn spawn(&self, principal: &Principal) -> FileSystemResult<Self> {
+        println!("spawn on: {:?}", principal);
```

Comment:
Are all the `println!`s added just temporary debug code that will be removed before we merge?

Reply:
It is temporary.